packages/main/src/plugin/telemetry/telemetry-api.ts

```diff
@@ -0,0 +1,36 @@
+/**********************************************************************
```

Comment:
Should this file be in the `packages/api` ?

Reply:
recommendations.json is at the root, and like in that case the next PR here will be to try to download a fresher file at runtime so I kept it in the same place. I don't have a strong opinion if anyone thinks it should move though.

Reply:
I'm okey with it